### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by torkleyy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,71 +1,113 @@
...
+// A component contains data
+// which is associated with an entity.
+
+#[derive(Debug)]
+struct Vel(f32);
+#[derive(Debug)]
+struct Pos(f32);
+
+impl Component for Vel {
+    type Storage = VecStorage<Vel>;
```
Comment (WaDelma) [inline README.md]: BTW you can use `VecStorage<Self>` here.

**Comment by notriddle** [comment]
@torkleyy You can add @kvark as a reviewer by going to your [repositories page](https://bors-app.herokuapp.com/repositories), then to slide-rs/specs, then to Settings.

**Comment by torkleyy** [comment]
@notriddle Thanks!
@kvark Should work now. Please try again.

**Pull request merged**